src/main/js/main_html5.js

```diff
@@ -873,11 +875,37 @@ require("../../../html5-common/js/utils/environment.js");
             lang: audioTracks[index].language,
             enabled: audioTracks[index].enabled
           };
+          if (audioTracks[index].enabled) {
+            currentAudioId = audioTracks[index].id;
+          }
           audioTrackList.push(element);
         }
-        return audioTrackList;
       }
-      return [];
+      return audioTrackList;
+    };
+
+    /**
+     * Sets the audio track to the ID specified by trackID
+     * @public
```

Comment:
Can we update the video plugin template with this new public function? The template acts as a form of documentation for video plugin developers.

Reply:
\+